### PR TITLE
Usage fix

### PR DESCRIPTION
**Pull request by majst01**
Usage number for availablePrefixes and availableIPs does not give any value if greater than int32, implementing in big.Int is a mess

### CODE REVIEW

File: prefix.go [hunk 1]
```diff
@@ -459,6 +462,10 @@ func (p *Prefix) availableips() uint64 {
 	if err != nil {
 		return 0
 	}
+	// We don't report more than 2^32 available IPs by design
+	if (ipprefix.IP.BitLen() - ipprefix.Bits) > 32 {
+		return math.MaxInt32
```
Comment (droid42) [inline prefix.go:467]: Do you want to cap to math.Max**U**int32, as this is 2^32 - 1

File: prefix.go [hunk 2]
```diff
@@ -495,6 +502,10 @@ func (p *Prefix) availablePrefixes() (uint64, []string) {
 		totalAvailable += 1 << (maxBits - pfx.Bits)
 		availablePrefixes = append(availablePrefixes, pfx.String())
 	}
+	// we are not reporting more that 2^32 available prefixes
+	if totalAvailable > math.MaxInt32 {
+		totalAvailable = math.MaxInt32
```
Comment (droid42) [inline prefix.go:507]: See above

**Review by droid42 — CHANGES_REQUESTED** [review]
Unsure if you wanted to cap at MaxUint32 or MaxInt32 (signed)

File: prefix.go (hunk 1 above)
Comment (majst01) [reply prefix.go:467]: Sadly swagger will only  create a int32 which will not be able to deserialize uint32 max.

Comment (droid42) [reply prefix.go:467]: Ah, ok, I think you said that already.

The only thing I would suggest then is to change "if (ipprefix.IP.BitLen() - ipprefix.Bits) > 32 {" to "if (ipprefix.IP.BitLen() - ipprefix.Bits) > **31** {" and comments to "2^31" to be correct. Or am I missing something?

Comment (majst01) [reply prefix.go:467]: Done

File: prefix.go (hunk 2 above)
Comment (majst01) [reply prefix.go:507]: also

**Review by droid42 — APPROVED** [review]
LGTM

**Pull request merged**